Source/Orts.Formats.Msts/CabViewFile.cs

```diff
@@ -845,7 +855,7 @@ public class CVCDiscrete : CVCWithFrames
         private int numPositions;
         private bool canFill = true;
 
-        public CVCDiscrete(STFReader stf, string basepath)
+        public CVCDiscrete(STFReader stf, string basepath, DiscreteStates discreteStates)
```

Comment:
The name of the parameter should be singular.

Reply:
Yes, of course it should.

Good catch.